app/[domain]/handle/page.tsx

```diff
@@ -0,0 +1,129 @@
+import { type Metadata } from "next"
+import { AppBskyActorDefs } from "@atproto/api"
+
+import { agent } from "@/lib/atproto"
+import { prisma } from "@/lib/db"
+import { Link } from "@/components/link"
+import { LoadMore } from "@/components/load-more"
+import { Profile } from "@/components/profile"
+
+export const revalidate = 3600
+
+interface Props {
+  params: { domain: string }
+}
+
+export async function generateMetadata({ params }: Props): Promise<Metadata> {
+  const domain = params.domain
+
+  return {
+    title: `The ${domain} Community`,
+    description: `See all the members of the ${domain} community.`,
+  }
+}
+
+const PAGE_SIZE = 100
+
+export default async function CommunityPage({ params }: Props) {
+  const domain = params.domain
+
+  const [count, { profiles: initialProfiles, nextOffset }] = await Promise.all([
+    prisma.user.count({
+      where: { domain: { name: domain } },
+    }),
+    getUsers(domain),
+  ])
+
+  return (
+    <main className="container grid items-center gap-6 pb-8 pt-6 md:py-10">
+      <div className="flex max-w-[980px] flex-col items-start gap-4">
+        <h1 className="text-3xl font-extrabold leading-tight tracking-tighter sm:text-3xl md:text-5xl">
+          Fellas with the {domain} Handle
+        </h1>
+        <p className="max-w-[500px] text-lg text-muted-foreground sm:text-xl">
+          Want to join the {count} fellas of the {domain} handle? Get your own
```

Comment:
Typo - The sub-heading should say "Want to join the [NumberOfMembers] fellas who are using the fellas.social handle? [Get your own](https://fellas.social/)"